lib/record_invocation/record_invocation.rb

```diff
@@ -53,25 +53,19 @@ def __one_invocation(method_name, **parameters)
   alias :one_invocation :__one_invocation
 
   def __invocations(method_name=nil, **parameters)
-    if method_name.nil? && parameters.empty?
-      return __records
-    end
+    invocations = __records
 
-    invocations = __records.select { |invocation| invocation.method_name == method_name }
-
-    if parameters.nil?
```

Comment:
@sbellware This is actually wrong -- `parameters` will always be a hash, and never nil. I replicated defect in my change. It's innocuous ultimately, but it's misleading.

Reply:
I updated mine to use `parameters.empty?`